src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacci.java

```diff
@@ -0,0 +1,58 @@
+package com.lftechnology.java.training.krishna.basics.even_fibonacci;
```

Comment:
I would much prefer if you avoid using `_` on the Package name. It is only used on special circumstances. 

Please refer section : Legalizing Package Names of below link : 

https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html